### DISCUSSION TITLE
Can you help me with fix some issues in repo mogul-christmas?

### DISCUSSION THREAD

**Discussion by KendallDoesCoding** [TechStudent10/techstudent10#7] [General]
Hi,

I need some more help in Javascript. 

Can you help me with this - https://github.com/KendallDoesCoding/mogul-christmas/issues/9?

Thanks,
Kendall

**Comment by KendallDoesCoding** [comment]
@TechStudent11 I made two more issues for enhancements and would like your help if you know how to do it

The issues are below:
1) [FEATURE REQUEST] Auto-embed in lyrics instead of iframe - https://github.com/KendallDoesCoding/mogul-christmas/issues/10
2) [FEAUTRE REQUEST] Const lyrics in Javascript - https://github.com/KendallDoesCoding/mogul-christmas/issues/11

**Reply by TechStudent10** [reply]
I'm sorry but I'm kinda busy right now. I'll come back to these later

**Reply by KendallDoesCoding** [reply]
Ok, take your time. The features arent needed on priority so that's fine. Thanks

**Reply by KendallDoesCoding** [reply]
I'm just horrible with JS that's why I asked for your help LOL. I really need to learn some Javascript soon!

**Reply by KendallDoesCoding** [reply]
When will you be able to help me @TechStudent11 ?

**Reply by TechStudent10** [reply]
@KendallDoesCoding Hopefully later today.

**Reply by KendallDoesCoding** [reply]
Ok cool. Thanks! 

Merry Christmas in advance!

**Reply by TechStudent10** [reply]
@KendallDoesCoding Same to you!

**Comment by KendallDoesCoding** [comment]
Yo @TechStudent11 ,

Just a reminder on this. I need you to check 5/7 issues i created in your free time and see if you can help.

Thanks,
@KendallDoesCoding

**Reply by KendallDoesCoding** [reply]
@TechStudent11 Sorry for bothering you so much. I know you must be really busy. Just these issues are really bothering me lol.

**Reply by TechStudent10** [reply]
@KendallDoesCoding I forgot about this. I'm gonna get started now.

**Comment by KendallDoesCoding** [comment]
Thanks a ton! I see the PR's

**Comment by KendallDoesCoding** [comment]
@TechStudent11 Thanks for your help with the issues so far. I need you to also have a look at this issue: https://github.com/KendallDoesCoding/mogul-christmas/issues/20

Thanks,
@KendallDoesCoding

**Reply by KendallDoesCoding** [reply]
@TechStudent11 Awaiting your response on this.

**Comment by KendallDoesCoding** [comment]
@TechStudent11 Can you help me with: 

https://github.com/KendallDoesCoding/mogul-christmas/issues/32
https://github.com/KendallDoesCoding/mogul-christmas/issues/20
https://github.com/KendallDoesCoding/mogul-christmas/issues/14

**Reply by TechStudent10** [reply]
@KendallDoesCoding sorry it's just that I was out all yesterday and couldn't check GitHub.

**Comment by KendallDoesCoding** [comment]
Also, can you help me with this?

@TechStudent11 can you make it in JS so that every year/when computer time is on date 25th it will display a p tag? rn what im doing is im doing some edits and deploying that on Netlify, but if its encoded into the website that would be pog.

**Reply by TechStudent10** [reply]
@KendallDoesCoding sorry it's just that I was out all yesterday and couldn't check GitHub.

**Comment by KendallDoesCoding** [comment]
@TechStudent11 Do you like the new christmas tree images in the bottom left & right? or do you think its un-necessary?

Please gimme your feedback

Site located here: https://mogulchristmas.kendalldoescoding.gq/

**Reply by TechStudent10** [reply]
Those look good!

**Reply by KendallDoesCoding** [reply]
Ok, thanks! I thought so too but I wanted some feedback on it lol, incase yall didnt like it.

**Comment by KendallDoesCoding** [comment]
@TechStudent11 Can you say something in this issue and let me know if you can do it or no, so I can assign you?

https://github.com/KendallDoesCoding/mogul-christmas/issues/60

**Reply by TechStudent10** [reply]
I don't understand what you want me to do.

**Reply by KendallDoesCoding** [reply]
First, I want to know if you can do this. And if yes, please comment on that issue so I can assign you to that issue because idk how to fix it.

**Comment by KendallDoesCoding** [comment]
Also do you have any idea with this?

https://github.com/KendallDoesCoding/mogul-christmas/issues/62
https://github.com/KendallDoesCoding/mogul-christmas/issues/60

**Comment by KendallDoesCoding** [comment]
Is this done? https://github.com/KendallDoesCoding/mogul-christmas/pull/66

I fixed the issue with embed not center aligning

**Reply by KendallDoesCoding** [reply]
@TechStudent11 For this, can you make it so that after the song gets over, it will go back to no embed again, until the user clicks on another song. That would be very cool.

**Comment by KendallDoesCoding** [comment]
@TechStudent11 Can I make it so that you will get auto-assigned to the JS file, when someone creates a PR related to the JS in mogul-christmas. Coz, idk anything about the js, you coded that mostly lol.

**Reply by KendallDoesCoding** [reply]
@TechStudent11 Waiting for your reply on this.

**Reply by TechStudent10** [reply]
ehhhhh I can't. it's not possible.

**Reply by KendallDoesCoding** [reply]
@TechStudent11 It is, and I'm not asking if you can do this, Ik how to do this, I'm not asking you to do it I wanna know if I should set it to auto-assign you to any PR's related with the .js file. Please let me know.

Thanks

**Reply by TechStudent10** [reply]
Huh? What? How? You can't auto-assign someone to a specific file.

**Reply by KendallDoesCoding** [reply]
Using the CODEOWNERS file easy. If you make a test pr in that repo, I auto get assigned, see for ya self.

**Reply by KendallDoesCoding** [reply]
I see need to say *.js @TechStudent11 , I can do it rn, but I want your permission first. The repo is mainly small, mostly you and me, but if ever anyone contributes to the JS, I think you will understand better coz idrk js that well so that's why I wanted to ask your permission first.

**Reply by KendallDoesCoding** [reply]
![image](https://user-images.githubusercontent.com/84712013/147768683-fd1c2d78-55c1-401d-9045-ca7f9a8f3db4.png)

**Reply by TechStudent10** [reply]
Okay, if you say so.

**Reply by KendallDoesCoding** [reply]
So, should I add you to be auto-assigned as the reviewer for js files or no @TechStudent11 ?

**Reply by TechStudent10** [reply]
yeah sure

**Reply by KendallDoesCoding** [reply]
Ok, will do so now.

**Reply by KendallDoesCoding** [reply]
Done!

**Reply by TechStudent10** [reply]
@KendallDoesCoding I am currently recording a video, so I am not gonna respond for a while.

**Reply by KendallDoesCoding** [reply]
Yeah, I can see from your status, for your YT?

**Reply by TechStudent10** [reply]
@KendallDoesCoding yup

**Reply by KendallDoesCoding** [reply]
Ok.. Also this says, you own the JS file now, but the reviewer thing doesn't seem to work, It may be cause I'm the owner of the repo... Still not sure...

**Comment by KendallDoesCoding** [comment]
When will you be able to help me with this @TechStudent11 - https://github.com/KendallDoesCoding/mogul-christmas/issues/38

**Comment by KendallDoesCoding** [comment]
Please have a look at

https://github.com/KendallDoesCoding/mogul-christmas/issues/77

**Reply by KendallDoesCoding** [reply]
Thanks for your help on this, but can you add the style="display:none" in the css itself?

https://github.com/KendallDoesCoding/mogul-christmas/issues/79

**Comment by KendallDoesCoding** [comment]
Also, I need help with this css pretty urgently lol


https://github.com/KendallDoesCoding/mogul-christmas/issues/80

**Comment by KendallDoesCoding** [comment]
I merged, https://github.com/KendallDoesCoding/mogul-christmas/pull/83 but I still need your help to make the darkmode toggle button small if the screen size is small. Please make a PR on that!

**Reply by KendallDoesCoding** [reply]
Waiting for your reply on this. Can you do this @TechStudent11 ?

**Reply by TechStudent10** [reply]
ohhh k. that might be hard 'cause that's not my code so it might take me a while to figure out how this works.

**Comment by KendallDoesCoding** [comment]
Yes. aliraza told it will be easy to be done and that you should do it. If
you can't then I'll go back to him and ask him to get it done.

**Reply by TechStudent10** [reply]
WHEN he's feeling better

**Comment by KendallDoesCoding** [comment]
Yes, exactly. Forgot to mention that

**Comment by KendallDoesCoding** [comment]
@TechStudent11 Why did you create this branch, I didnt wanna revert this change. https://github.com/Kendall-Does-Coding-Websites/mogul-christmas/tree/revert-140-testings

**Reply by TechStudent10** [reply]
I didn't mean to, I was curious. I assumed it didn't get created. Sorry!

**Reply by KendallDoesCoding** [reply]
Ok, no problem. I'll delete the branch.

**Comment by KendallDoesCoding** [comment]
I mentioned you in this PR please check it out: https://github.com/KendallDoesCoding/mogul-christmas/pull/150

**Comment by KendallDoesCoding** [comment]
Hi @TechStudent11 ,

Please get back to my comment on https://github.com/KendallDoesCoding/mogul-christmas/pull/207 when you get time :).. Thanks

> @TechStudent11 Please have a look now after Github Copilot's suggestions made to the code.
> Tbh, I don't really like the icon near the button now, I think the button looks better alone. What do yall think about it @TechStudent11 @sc0rp10n-py ?

**Comment by KendallDoesCoding** [comment]
@TechStudent11 Please have a look at https://github.com/KendallDoesCoding/mogul-christmas/issues/205

> WOW. This is actually easy to get done... @TechStudent11 Can I work on this if you haven't already started working on it, that is. https://stackoverflow.com/questions/24509404/how-do-i-open-up-a-random-url-from-a-list-of-urls-in-a-new-time-each-time-it-is

**Comment by KendallDoesCoding** [comment]
Hey @TechStudent11 ,

I need you to work on this, as soon as possible, I need this done within the next day or two, if possible. : https://github.com/KendallDoesCoding/mogul-christmas/issues/288.

And then, need you to work on this, but there's no rush on this, you can work on this, end of April, early May too.: https://github.com/KendallDoesCoding/mogul-christmas/issues/276.

Thanks a ton,
KendallDoesCoding
